src/website2/Dockerfile

```
@@ -1,33 +1,36 @@
-ENV NEXT_PUBLIC_API_URL=$NEXT_PUBLIC_API_URL \
-    NEXT_PUBLIC_OPENCAGE_API_KEY=$NEXT_PUBLIC_OPENCAGE_API_KEY \
-    NEXT_PUBLIC_API_TOKEN=$NEXT_PUBLIC_API_TOKEN
+# Create a non-root user for security
+RUN addgroup -S app && adduser -S -G app app
 
-# Build the Next.js application for production
-RUN npm run build
+# Copy node_modules, package.json, built output, and public assets
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/package.json ./package.json
+COPY --from=builder /app/.next ./.next    # full build output :contentReference[oaicite:1]{index=1}
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)